lib/src/utils/flags.dart

```diff
@@ -22,6 +22,12 @@ class Flags<T extends Flags<T>> extends IterableBase<Flag<T>> with ToStringHelpe
   /// Return a set of flags that has all the flags set in both `this` and [other].
   Flags<T> operator &(Flags<T> other) => Flags(value & other.value);
 
+  /// Return a set of flags that has all the flags set in `this` but not in [other].
```

Comment:
That's not the correct descriription of XOR - it should be something like this:
```suggestion
  /// Return a set of flags that has all the flags set in `this` or in `other` but not in both.
```